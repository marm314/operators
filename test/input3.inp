   +1 acp aaq acr aas acm aan
   -1 acr aas acm aan acp aaq
