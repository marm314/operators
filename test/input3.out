   +1 acp acr aas aan 	* delta_mq
   +1 acm acp aas aan 	* delta_qr
   +1 acp aan 	* delta_msdelta_qr
   -1 acm acr aaq aan 	* delta_ps
   -1 acm acr aas aaq 	* delta_np
   -1 acr aaq 	* delta_msdelta_np
